vNext/src/Orleans/Shims/AppDomain.cs

```diff
@@ -38,6 +38,10 @@ public Assembly[] GetAssemblies()
             {
                 typeof(Exception).GetTypeInfo().Assembly,
                 typeof(AssemblyProcessor).GetTypeInfo().Assembly,
+                // load up classes for testing, until Assembly scanning is supported in vNext
+                Assembly.Load(new AssemblyName("Orleans.NonSiloTests")),
```

Comment:
is Orleans.NonSiloTests required? because that's my only concern with this approach. Still, if it is indeed required we can still merge for now and then fix in when we have more test projects, so it doesn't break those.